src/main/resources/standardTestCases/exit.test

```diff
@@ -0,0 +1,13 @@
+name: exit
+testFormat: workflowsuccess
+backends: [Jes, Local]
```

Comment:
@Horneth So this is just a filter - it will only allow a test to run on these backends. For instance you could use this if you had a docker-less test to only allow it to run on Local. Beyond, it works like anything else - backends can be specified in options, etc but otherwise runs on default.

So for this example it'll always run on default which means both local and jes variants in our travis builds